docs/terms/validation_result.md

```diff
@@ -1,15 +1,91 @@
+
+### Relationship to other objects
+
+Validation Results are generated inside a Checkpoint when a <TechnicalTag relative="../" tag="validator" text="Validator" /> runs an Expectation Suite against its paired <TechnicalTag relative="../" tag="batch_request" text="Batch Request" />.  The Validation Result will be passed to the Checkpoint's `action_list` where it will be available to any Actions that may need it.  The `StoreValidationResultAction` subclass of `ValidationAction` will save Validation Results to the <TechnicalTag relative="../" tag="validation_result_store" text="Validation Result Store" /> if it is present in a Checkpoint's `action_list`.
+
+## Use cases
+
+<CreateHeader/>
+
+A Validation Result is generated when you use [the interactive workflow to create Expectations](../guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md) and the newly created Expectation is run against the sample <TechnicalTag relative="../" tag="batch" text="Batch" /> of data you are working with.  These individual Validation Results will be created as an `ExpectationValidationResult` instance.
+
+<ValidateHeader/>
+
+Validation Results are generated when you run Checkpoints to Validate data.  Because Checkpoints operate on Expectation Suites rather than individual Expectations, the Validation Results generated by Checkpoints will be created as `ExpectationSuiteValidationResult` instances. The difference between an `ExpectationValidationResult` instance and an `ExpectationSuiteValidationResult` instance is that the `ExpectationSuiteValidationResult` will contain additional information pertaining to the Expectation Suite that was Validated, and will contain the data of an `ExpectationValidationResult` for each Expectation in the Expectation Suite.  These `ExpectationValidationResult` instances will be in a list accessed by the `results` attribute of the `ExpectationSuiteValidationResult` instance.
+
+The Validation Results generated as `ExpectationSuiteValidationResult` instances can be saved in JSON format in a Validation Result Store, and rendered as <TechnicalTag relative="../" tag="data_docs" text="Data Docs" />.  Both of these activities are handled by Actions in the Checkpoint's `action_list`.  In most cases, when you see "Validation Results" in Great Expectations, it will be in reference to the output generated as an `ExpectationSuiteValidationResult` when a Checkpoint is run.
+
+## Features
+
+### Trends in data quality
+
+Saved Validation Results allow you to track trends in how well your data adheres to Expectations.  This can help you determine if failed Expectations are due to outliers in new data, or a more systemic problem.
```

Comment:
This is super interesting and I dont think I thought about Validation results in this way before. I'm curious about how this works in OSS vs Cloud. 

Would the OSS version be something like: (sorry for the overly simple example) "if one of your Expectations has been failing every time for the last week, something is probably systematically wrong with your data"?

Reply:
One example of this for me would be for illustrating data drift. Maybe you have `expect_column_mean_to_be_between(min_value=0, max_value=100)`. The first week, the mean might be 10, the next week 50, the next 100, the next 140.  Only the last one would fail, but the EVRs would tell a compelling story.

Overall, I love this use case, but we don't currently have a great way of actually doing this internally. Users who do this tend to parse the JSON and pass those results to a dashboard. I'm not sure whether or not that matters for the purposes of this doc.